src/utils.ts

```diff
@@ -22,4 +22,28 @@ const uploadImage = async (imageB64: string): Promise<string> => {
   return responseData.data;
 };
 
-export default uploadImage;
+/**
+ * Remove an image from AppDev servers
+ *
+ * @param imageURL the image URL to remove
+ * @returns true if the deletion was successful; otherwise false
+ */
+const removeImage = async (imageURL: string): Promise<boolean> => {
+  // Delete image via a POST request
+  const payload = {
+    bucket: UPLOAD_BUCKET,
+    image_url: imageURL,
+  };
+  const response = await fetch(`${process.env.UPLOAD_SERVICE_URL}/remove/`, {
+    method: 'POST',
+    body: JSON.stringify(payload),
+  });
+
+  const responseData = JSON.parse(await response.text());
```

Comment:
```suggestion
  const responseData = await response.json();
```

Reply:
I tried doing this before but the issue is that I want to access the `success` propery of the JSON. Using `response.json()` will give `responseData` an unknown type so Typescript will say "Property `success` does not exist on type `unknown`".

Reply:
Yes what you should do then is define a type that specifies what the response JSON will look like and specify that that is the type of the JSON response data. You can checkout an example of this in one of my repos. I define the type interface [here](https://github.com/Archit404Error/SemanticClassSearch/blob/main/mobile/index.d.ts) and make the request [here](https://github.com/Archit404Error/SemanticClassSearch/blob/8d1db8c34490bfbe428080ff4a43802640e54d5c/mobile/app/(tabs)/search/index.tsx#L9-L14)

Reply:
I created an interface at the top of `utils.ts` called `UploadResponse` and wrote the following:
```
const responseData = (await response.json()) as UploadResponse;
if (!responseData.success) {
  console.log(`Removing an image from our servers failed with image URL: ${imageURL}`);
}
return responseData.success;
```
This works as intended, but do you want to have a `.d.ts` file instead?

Reply:
The interface should be fine for now, we can always pull this out into a .d.ts file later

Reply:
You might also be able to just do `const responseData : UploadResponse = await response.json()`. Once you commit this I'll approve

Reply:
`const responseData : UploadResponse = await response.json()` doesn't work. I believe it's because the response has an `undefined` type. I kept what I wrote earlier and everything works perfectly for adding and deleting flyers.